PyRDF/CallableGenerator.py

```diff
@@ -66,30 +61,27 @@ def get_callable(self):
 
         self.root_node.graph_prune()
 
-        self.actions = 0
-        self.tfs = 0
-        self.operations = []
-        self.action_node_map = {}
-        self._dfs(self.root_node)
+        def mapper(node_cpp, node_py=None, return_vals=[], return_nodes=[]):
```

Comment:
I am confused... why do you still `return_vals` and `return_nodes` here?

Reply:
@etejedor , yes. Sorry that I forgot to remove those. I'll fix it in my next commit